app/components/post_list/post/body/add_members/add_members.tsx

```diff
@@ -81,11 +81,11 @@ const definedMessages = defineMessages({
     },
     messageOne: {
         id: 'post_body.check_for_out_of_channel_mentions.message.one',
-        defaultMessage: '{username} was mentioned but is not in the channel. Would you like to ',
+        defaultMessage: 'was mentioned but is not in the channel. Would you like to ',
```

Comment:
Why the removal of the variable here and the line below?

Reply:
Same answer.

If you look further down at the implementation, you see that the formatted texts are not passing any values, so `username` would never be populated either way. In this case this is the correct value (unless we want to refactor the whole thing to be more translation friendly, but that is out of the scope right now).